ngrok/src/config/http.rs

```diff
@@ -137,6 +137,14 @@ impl TunnelConfig for HttpOptions {
             .clone()
             .unwrap_or(default_forwards_to().into())
     }
+
+    fn forwards_proto(&self) -> String {
+        self.common_opts
+            .forwards_proto
+            .clone()
+            .unwrap_or(String::new())
```

Comment:
clippy will probably yell about this. If it does, `unwrap_or_default` instead. If not, leave it :stuck_out_tongue: